openmetadata-ui/src/main/resources/ui/src/axiosAPIs/topicsAPI.ts

```diff
@@ -128,7 +128,7 @@ export const patchTopicDetails = async (id: string, data: Operation[]) => {
 
 export const restoreTopic = async (id: string) => {
   const response = await APIClient.put<
-    RestoreEntitiesRequestType,
+    RestoreRequestType,
```

Comment:
Please verify this changes and make sure that it's not breaking anything existing